client/apps/settings/stylesheets/index.styl

```diff
@@ -28,3 +28,49 @@
     color gray-darker-color
   &:last-of-type
     border-bottom none
+
+.page-header.sticky
+  position fixed
+  top 0
+  left 110px
+  right 0
+  background-color white
+  z-index 100
+
+.edit-feature
```

Comment:
Minor but ideally this is namespaced by the containing app, e.g. `.settings-edit-feature`.